cmd/earthly/main.go

```diff
@@ -1205,9 +1309,30 @@ func (app *earthlyApp) configureSatellite(cc cloud.Client) error {
 	app.console.Warnf("Note: the Interactive Debugger, Interactive RUN commands, and Local Registries do not yet work on Earthly Satellites.")
 
 	// Set up extra settings needed for buildkit RPC metadata
-	app.buildkitdSettings.BuildkitAddress = app.satelliteAddress
+	app.satelliteName = app.cfg.Satellite.Name
+	app.satelliteOrg = app.cfg.Satellite.Org
+	orgID, err := app.getSatelliteOrgID(cc)
+	if err != nil {
+		return err
+	}
 	app.buildkitdSettings.SatelliteName = app.cfg.Satellite.Name
-	app.buildkitdSettings.SatelliteOrg = app.cfg.Satellite.Org
+	app.buildkitdSettings.SatelliteOrgID = orgID
+	if app.satelliteAddress != "" {
+		app.buildkitdSettings.BuildkitAddress = app.satelliteAddress
+	} else {
+		app.buildkitdSettings.BuildkitAddress = containerutil.SatelliteAddress
+	}
+
+	app.console.Warnf("") // newline
+	app.console.Warnf("The following feature flags are recommended for use with Satellites and will be auto-enabled:")
+	app.console.Warnf("--new-platform, --use-registry-for-with-docker")
+	app.console.Warnf("") // newline
+
+	if app.featureFlagOverrides == "" {
+		app.featureFlagOverrides = "new-platform,use-registry-for-with-docker"
```

Comment:
This seemed like a clean way to set these feature flags from the central `configureSatellite` function but let me know if there's a more preferred method of doing it.

Reply:
seems like a reasonable place to set them.

Can you change it to something like:

```
if app.featureFlagOverrides != "" {
    app.featureFlagOverrides += ","
}
app.featureFlagOverrides += "new-platform,use-registry-for-with-docker"
```

so we don't have the flags copy/pasted in both parts of the if/else?

Reply:
Yep that is cleaner 👍